build.gradle.kts

```diff
@@ -58,3 +63,59 @@ jsonSchema2Pojo {
 	// What type to use instead of string when adding string properties of format "date-time" to Java types
 //	dateTimeType = "java.time.LocalDateTime"
 }
+
+val jacocoExclude = listOf("**/generated/**",
+		                   "**/api/**",
+		                   "**/cma/**")
```

Comment:
A minor point, but the indentation is inconsistent on these 3 lines. Presumably there are tools that can auto-format this file. Is there a tool that the Java community here use, that we might adopt as a team?

Reply:
this looks fine in InteliJ 
<img width="503" alt="Screenshot 2023-06-30 at 09 17 52" src="https://github.com/ministryofjustice/cfe-crime/assets/137802184/c92f817f-e51a-4e4b-b4d6-89e69e9a9ee8">

Reply:
Everything on one line. On Intellij cannot changes tabs to spaces or vice versa as this is a standard copy.